packages/pretty-format/src/plugins/DOMElement.ts

```diff
@@ -82,11 +82,14 @@ export const serialize: NewPlugin['serialize'] = (
             .map(attr => attr.name)
             .sort(),
       nodeIsFragment(node)
-        ? []
-        : Array.from(node.attributes).reduce((props, attribute) => {
-            props[attribute.name] = attribute.value;
-            return props;
-          }, {} as any),
+        ? {}
```

Comment:
*This* could actually be classed as a bug - it might not ever be reachable due to surrounding code, but still - it was an array when it should have been an object 🤷‍♂